TestFlightAPI/TestFlightAPI/TestFlightReliability.cs

```diff
@@ -254,17 +254,32 @@ public virtual List<string> GetTestFlightInfo()
                 return infoStrings;
             }
 
+            double currentFailRate = core.GetBaseFailureRate();
+            double maxFailRate = GetBaseFailureRate(reliabilityCurve.maxTime);
+
             infoStrings.Add("<b>Base Reliability</b>");
-            infoStrings.Add(String.Format("<b>Current Reliability</b>: {0} <b>MTBF</b>", core.FailureRateToMTBFString(core.GetBaseFailureRate(), TestFlightUtil.MTBFUnits.SECONDS, 999)));
-            infoStrings.Add(String.Format("<b>Maximum Reliability</b>: {0} <b>MTBF</b>", core.FailureRateToMTBFString(GetBaseFailureRate(reliabilityCurve.maxTime), TestFlightUtil.MTBFUnits.SECONDS, 999)));
+            infoStrings.Add(String.Format("<b>Current Reliability</b>: {0:P1} at full burn, {1} <b>MTBF</b>", TestFlightUtil.FailureRateToReliability(currentFailRate, configBurnTime), core.FailureRateToMTBFString(currentFailRate, TestFlightUtil.MTBFUnits.SECONDS, 999)));
+            infoStrings.Add(String.Format("<b>Maximum Reliability</b>: {0:P1} at full burn, {1} <b>MTBF</b>", TestFlightUtil.FailureRateToReliability(maxFailRate, configBurnTime), core.FailureRateToMTBFString(maxFailRate, TestFlightUtil.MTBFUnits.SECONDS, 999)));
 
             return infoStrings;
         }
 
-        public virtual string GetModuleInfo()
+        public virtual string GetModuleInfo(string configuration, float configBurnTime)
         {
             return string.Empty;
         }
+        public virtual float GetRatedBurnTime(string configuration)
+        {
+            return 0f;
+        }
+        public virtual float GetRatedBurnTime()
+        {
+            return 0f;
+        }
+        public virtual float GetCurrentBurnTime()
+        {
+            return 0f;
+        }
```

Comment:
These really feel out of place in a generic base module.

Reply:
Agreed, would you prefer it if TestFlightCore did some type based shenanigans instead? Or is that just more clunky.